I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00keymap.rpy", line 388, in script
    python hide:
  File "renpy/common/00keymap.rpy", line 403, in <module>
    renpy.save("_reload-1", "reload save game")
AttributeError: 'NoneType' object has no attribute 'stores'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00keymap.rpy", line 388, in script
    python hide:
  File "C:\Users\linno\Desktop\renpy-6.99.12.2-sdk\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\linno\Desktop\renpy-6.99.12.2-sdk\renpy\python.py", line 1695, in py_exec_bytecode
    exec bytecode in globals, locals
  File "renpy/common/00keymap.rpy", line 403, in <module>
    renpy.save("_reload-1", "reload save game")
  File "C:\Users\linno\Desktop\renpy-6.99.12.2-sdk\renpy\loadsave.py", line 280, in save
    roots = renpy.game.log.freeze(None)
  File "C:\Users\linno\Desktop\renpy-6.99.12.2-sdk\renpy\python.py", line 1592, in freeze
    self.complete()
  File "C:\Users\linno\Desktop\renpy-6.99.12.2-sdk\renpy\python.py", line 1269, in complete
    self.current.stores[name] = sd.get_changes()
AttributeError: 'NoneType' object has no attribute 'stores'

Windows-8-6.2.9200
Ren'Py 6.99.12.2.2029
 
